repos.d/izzyondroid.yaml

```diff
@@ -0,0 +1,26 @@
+###########################################################################
+# IzzyOnDroid andriod repository
+###########################################################################
+- name: izzyondroid
+  type: repository
+  desc: IzzyOnDroid F-Droid Repository
+  family: fdroid
+  ruleset: izzyondroid
+  minpackages: 10
+  sources:
+    - name: index.xml
+      fetcher:
+        class: FileFetcher
+        url: https://apt.izzysoft.de/fdroid/repo/index.xml
+      parser:
+        class: FDroidParser
+  shadow: true
+  repolinks:
+    - desc: IzzyOnDroid
+      url: https://apt.izzysoft.de/fdroid/index.php
+  packagelinks:
+    - type: PACKAGE_HOMEPAGE
+      url: 'https://apt.izzysoft.de/fdroid/index/apk/{srcname}'
+    - type: PACKAGE_RECIPE
+      url: 'https://apt.izzysoft.de/fdroid/index/info' # "Applications in this repository are official binaries built by the original application developers, taken from their resp. repositories (mostly Github)."
```

Comment:
Adding the same link to each package has no sense. You can add it to `repolinks` instead.